### PR TITLE
docs(dotnet): TestRunParameters -> Playwright runsettings node

### LINKED ISSUE #2215: [internal] List all the TestRunParameters
Relates #2214

### PR DESCRIPTION
**Pull request by mxschmitt**
Fixes https://github.com/microsoft/playwright-dotnet/issues/2215.

### CODE REVIEW

File: docs/src/running-tests-csharp.md [hunk 1]
```diff
@@ -29,20 +29,10 @@ You can run a single test, a set of tests or all tests. Tests can be run on diff
   dotnet test --filter "Name~TestMethod1"
   ```
 
-- Running Tests on specific browsers
+- Running Tests on specific browsers (either via the `BROWSER` env var or via the settings override)
```
Comment (kblok) [inline docs/src/running-tests-csharp.md:32]: I would move the clarification down bellow and leave an `env var` example

File: docs/src/test-runners-csharp.md [hunk 2]
```diff
@@ -383,32 +374,34 @@ When you have enabled the [verbose API log](./debug.md#verbose-api-logs), via th
 
 ### Using the .runsettings file
 
-When running tests from Visual Studio, you can take advantage of the `.runsettings` file.
+When running tests from Visual Studio, you can take advantage of the `.runsettings` file. The following shows a reference of the supported values.
 
-For example, to specify the amount of workers (`MSTest.Parallelize.Workers`), you can use the following snippet:
+For example, to specify the amount of workers you can use `MSTest.Parallelize.Workers` or to enable `DEBUG` logs `RunConfiguration.EnvironmentVariables`.
```
Comment (kblok) [inline docs/src/test-runners-csharp.md:379]:
```suggestion
For example, to specify the number of workers, you can use `MSTest.Parallelize.Workers`. You can also enable `DEBUG` logs using `RunConfiguration.EnvironmentVariables`.
```

**Review by kblok — APPROVED** [review]
🚀

File: docs/src/test-runners-csharp.md [hunk 3]
```diff
@@ -154,24 +154,18 @@ CLI. See the following example:
...
 <?xml version="1.0" encoding="utf-8"?>
 <RunSettings>
-  <TestRunParameters>
-    <Parameter name="browser" value="chromium" />
-    <Parameter name="headless" value="false" />
-    <Parameter name="channel" value="msedge" />
-  </TestRunParameters>
+  <Playwright>
+    <Browser>chromium</Browser>
+    <LaunchOptions>
```
Comment (yury-s) [inline docs/src/test-runners-csharp.md:159]: The more a look at this the more it feels that LaunchOptions should be under Browser.

**Review by yury-s — APPROVED** [review]

**Pull request merged**